packages/core/truffle-config.js

```diff
@@ -1 +1,6 @@
-module.exports = require("@uma/common").getTruffleConfig(__dirname);
+const config = require("@uma/common").getTruffleConfig(__dirname);
+
+// The default is 199. FiXME - comment out this line to see what goes wrong:
+config.compilers.solc.settings.optimizer.runs = 25;
```

Comment:
Which contracts now go over the bytecode limit? We typically remove require messages in the affected contracts rather than tweak this flag globally.